flytekit/bin/entrypoint.py

```diff
@@ -137,7 +137,41 @@ def _dispatch_execute(
             logger.warning("Task produces no outputs")
             output_file_dict = {_constants.OUTPUT_FILE_NAME: _literal_models.LiteralMap(literals={})}
         elif isinstance(outputs, _literal_models.LiteralMap):
-            output_file_dict = {_constants.OUTPUT_FILE_NAME: outputs}
+            offloaded_literals: Dict[str, _literal_models.Literal] = {}
+            literal_map_copy = {}
+
+            min_offloaded_size = int(os.environ.get("FK_L_MIN_SIZE_MB", "10")) * 1024 * 1024
+            max_offloaded_size = int(os.environ.get("FK_L_MAX_SIZE_MB", "1000")) * 1024 * 1024
```

Comment:
Do we use the same names here that we use in propeller
https://github.com/unionai/flyte/blob/c10b55373e9dea2bcce493f978499648c3f89d8f/flytepropeller/pkg/controller/config/config.go#L204-L207

Reply:
we'll need to update the [injection of environment variables ](https://github.com/flyteorg/flyte/blob/master/flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go#L141)to match those names.

The need for short env vars is described https://github.com/flyteorg/flyte/issues/5665.

Reply:
Ok with keeping the current ones .